components/server/src/container-module.ts

```diff
@@ -203,8 +166,16 @@ export const productionContainerModule = new ContainerModule((bind, unbind, isBo
 
     bind(TermsProvider).toSelf().inSingletonScope();
 
+    const contentServiceClient = new ContentServiceClient("content-service:8080", grpc.credentials.createInsecure())
```

Comment:
Maybe pedantic, but I'd love to see the `address` as part of the env with this value as default, even if not used yet by helm.